snuba/datasets/configuration/events_analytics_platform/entities/eap_spans.yaml

```diff
@@ -54,7 +57,13 @@ storages:
 storage_selector:
   selector: DefaultQueryStorageSelector
 
-query_processors: []
+query_processors:
```

Comment:
Can we get rid of this? I don't like this processor and would prefer it go away. It does a lot more than provide a `time` alias that this entity doesn't need.

Reply:
The TimeSeriesProcessor has been pretty helpful in making these queries easy to write, will on the performance team asked for it as well. What is the concern with it?